draft-ietf-suit-manifest.md

```diff
@@ -548,7 +556,14 @@ When the manifest processor encounters any of these scenarios the parallel proce
 * Set Strict Order = True.
 * Set Component Index.
 
-To perform more useful parallel operations, a manifest author may collect sequences of commands in a Run Sequence command. Then, each of these sequences MAY be run in parallel. Each sequence defaults to Strict Order = True. To isolate each sequence from each other sequence, each sequence MUST begin with a Set Component Index directive with the following exception: when the index is either True or an array of indices, the Set Component Index is implied. Any further Set Component Index directives MUST cause an Abort. This allows the interpreter that issues Run Sequence commands to check that the first element is correct, then issue the sequence to a parallel execution context to handle the remainder of the sequence.
+To perform more useful parallel operations, a manifest author may collect sequences of commands in a Run Sequence command. Then, each of these sequences MAY be run in parallel. There are several invocation options for Run Sequence:
+
+* Component Index is a positive integer, Strict Order is False: Strict Order is set to True before the sequence argument is run. The sequence argument MUST begin with set-component-index.
+* Component Index is True or an array of positive integers, Strict Order is False: The sequence argument is run once for each component (or each component in the array) the component index is set by the interpreter without a directive, and Strict Order is set to True before each iteration.
```

Comment:
the CDDL has "true" not "True" for Component Index, should this be "true" like in line 1224?
```suggestion
* Component Index is true or an array of positive integers, Strict Order is False: The sequence argument is run once for each component (or each component in the array) the component index is set by the interpreter without a directive, and Strict Order is set to True before each iteration.
```
Also I am having a hard time parsing grammar around the phrase "the component index is set by the interpreter without a directive", can you reword?  Same in 564